front/src/common/redux/reducers/loginReducer.js

```diff
@@ -0,0 +1,29 @@
+import * as types from '../actions/types';
+
+const initialState = {
+  isLogin: false,
+  errorMessage: '',
+};
+
+export function login(state = initialState, action) {
+  
+  const { response } = action;
+
+  switch(action.type) {
+    case types.LOGIN_USER_SUCCESS:
+      if(response) {
+        if (response.token) {
```

Comment:
It is better to implement it in the way, that if we get `types.LOGIN_USER_SUCCESS` action we handle success case, no error messages etc. Because if we have an error message it means that login is failed and we need to use another action

Reply:
types.LOGIN_USER_SUCCESS means success of sending user data, do I need rename this type like types.LOGIN_USER_CHECK ?